test/WebJobs.Script.Tests.Integration/Host/StandbyManagerTests.cs

```diff
@@ -36,7 +36,7 @@ public class StandbyManagerTests : IDisposable
         private TestLoggerProvider _loggerProvider;
         private HttpClient _httpClient;
         private TestServer _httpServer;
-        private string _expectedHostId;
+        private string _expectedHostId = "update when tests fixed";
```

Comment:
Setting this temporarily to address the assignment warning. Once the actual tests are updated, this will be set to the expected value.